migration-v4.md

```diff
@@ -325,6 +325,7 @@ module.exports = {
 ```
 
 - Default value of the `static` option is `path.resolve(process.cwd(), 'public')` directory and enabled by default.
+- Watching of static files is enabled by default.
```

Comment:
```suggestion
-  `static.watch` is enabled by default.
```

Reply:
nice

Reply:
I think we have an extra space at the beginning now, can you confirm?

Reply:
nah it looks fine